shelley/chain-and-ledger/executable-spec/test/Rules/ClassifyTraces.hs

```diff
@@ -38,13 +38,17 @@ relevantCasesAreCovered = withTests 500 $ property $ do
         (traceLength tr <= 5 * length (filter isRegKey certs_))
 
   cover 75
-        "there is at least 1 DeRegKey certificate for every 5 transactions"
-        (traceLength tr <= 5 * length (filter isDeRegKey certs_))
+        "there is at least 1 DeRegKey certificate for every 20 transactions"
+        (traceLength tr <= 20 * length (filter isDeRegKey certs_))
```

Comment:
I've modified this because the trace coverage tests were failing as we added new DCerts and thus proportionally decreased the % of `DeRegKey`s.